birdhouse/components/cowbird/default.env

```diff
@@ -50,6 +50,12 @@ export COWBIRD_LOG_LEVEL=INFO
 # Subdirectory of DATA_PERSIST_SHARED_ROOT containing the user workspaces used by Cowbird
 export USER_WORKSPACES="user_workspaces"
 
+THIS_FILE="`realpath "$0"`"
+# Override JupyterHub Readme file to use Cowbird's own readme version,
```

Comment:
Override, so now we have 2 similar copies that will have to be kept in sync?

Can't we instead add an extra readme file when cowbird is enabled to further explain its capabilities?

Reply:
I'm open to making it an extra separate readme. 
This readme should stay specific to Cowbird.

I don't know if @fmigneault on this, since I discussed this with him?

Reply:
Opps, sorry I forgot our Ouranos README is outside of the birdhouse-deploy repo, here https://github.com/Ouranosinc/PAVICS-landing/blob/master/content/jupyter-readme/README.ipynb.  We even have the matching French flavor and so we deploy both of them.

The assumption was that each organization will deploy their own README so there is not a one size-fit-all readme.

This is really annoying, we can not even try a "nb join" commands since we are missing the French translation.

Our structure, we have the additional `public/`, `mypublic/`, and `pavics-homepage/` dir.

Not sure what to do here.  I'll take @fmigneault proposal for now:
```shell
export JUPYTERHUB_COWBIRD_README="${COMPOSE_DIR}/components/cowbird/user_workspace_readme.ipynb"
export JUPYTERHUB_README="${JUPYTERHUB_README:-${JUPYTERHUB_COWBIRD_README}}"
```

Reply:
I updated the code with @fmigneault's proposal for now. At least, it will prevent overriding a previously existing definition of the `JUPYTERHUB_README` variable.